tests/Doctrine/Tests/TestUtil.php

```diff
@@ -61,16 +61,28 @@ private static function hasRequiredConnectionParams()
             $GLOBALS['db_type'],
             $GLOBALS['db_username'],
             $GLOBALS['db_password'],
-            $GLOBALS['db_host'],
-            $GLOBALS['db_name'],
-            $GLOBALS['db_port']
+            $GLOBALS['db_name']
+        )
+        && (
+            isset(
+                $GLOBALS['db_host'],
+                $GLOBALS['db_port']
+            ) || isset(
+                $GLOBALS['db_server']
```

Comment:
I think this is not true. `server` alone is not sufficient. I remember when implementing SQL Anywhere, the `server` parameter was sometimes required but is not sufficient without `host` and `port`. Because it is another abstraction beneath `host`. Not sure why `server` parameter is not in here though because I know that I used it for testing purposes and it worked,